plugins/hydra_ax/hydra_plugins/ax_sweeper/ax_sweeper.py

```diff
@@ -0,0 +1,275 @@
+            parameters[-1]["name"] = key
+        commandline_params = self.parse_commandline_args(arguments)
+        for cmd_param in commandline_params:
+            for param in parameters:
+                if param["name"] == cmd_param["name"]:
+                    for key, value in cmd_param.items():
+                        param[key] = value
+        ax_client = AxClient(
+            verbose_logging=self.verbose_logging, random_seed=self.random_seed
+        )
+        ax_client.create_experiment(parameters=parameters, **self.experiment)
+
+        return ax_client
+
+    def parse_commandline_args(self, arguments):
+        """Method to parse the command line arguments and convert them into Ax parameters"""
+
+        def _is_int(string_inp):
+            """Method to check if the given string input can be parsed as integer"""
+            try:
+                int(string_inp)
+                return True
+            except ValueError:
+                return False
+
+        def _is_float(string_inp):
+            """Method to check if the given string input can be parsed as a float"""
+            try:
+                float(string_inp)
+                return True
+            except ValueError:
+                return False
+
+        parameters = []
+        for arg in arguments:
+            key, value = arg.split("=")
+            if "," in value:
+                # This is a Choice Parameter.
+
+                value_choices = [x for x in value.split(",")]
+                if _is_float(value_choices[0]):
+                    parameter_type = ParameterType.FLOAT
+                    value_choices = [float(x) for x in value_choices]
+                else:
+                    parameter_type = ParameterType.STRING
+                parameters.append(
+                    {
+                        "name": key,
+                        "type": "choice",
+                        "values": value_choices,
+                        "parameter_type": parameter_type,
+                    }
+                )
+            elif ":" in value:
+                # This is a Range Parameter.
+                range_start, range_end = value.split(":")
+                if _is_int(range_start) and _is_int(range_end):
+                    range_start = int(range_start)
+                    range_end = int(range_end)
+                    param_type = ParameterType.INT
+                else:
+                    range_start = float(range_start)
+                    range_end = float(range_end)
+                    param_type = ParameterType.FLOAT
+                parameters.append(
+                    {
+                        "name": key,
+                        "type": "range",
+                        "bounds": [range_start, range_end],
+                        "parameter_type": param_type,
+                    }
+                )
+            else:
+                # This is a Fixed Parameter.
+                parameters.append(
```

Comment:
did you verify that fixed is working? I remember running into some exception when I tried.

Reply:
Yeah this is fixed now